.travis.yml

```diff
@@ -39,6 +44,7 @@ before_install:
 install:
     - travis_wait pip install -r requirements.txt
     - pip install sip pyqt5==5.11.*
+    - source $TRAVIS_BUILD_DIR/.travis/install_orange.sh
```

Comment:
Orange is (unnecessarily) installed already 2 lines above when installing requirements.txt
This new install_orange script then uninstalls it and installs the selected version (e.g. master)
You could try moving this up, if it still works correctly in that order, and avoid some extra work.